multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakSettings.java

```diff
@@ -53,14 +53,26 @@
   @Inject
   public KeycloakSettings(
       @Nullable @Named(JS_ADAPTER_URL_SETTING) String jsAdapterUrl,
-      @Named(AUTH_SERVER_URL_SETTING) String serverURL,
-      @Named(REALM_SETTING) String realm,
+      @Nullable @Named(AUTH_SERVER_URL_SETTING) String serverURL,
+      @Nullable @Named(REALM_SETTING) String realm,
       @Named(CLIENT_ID_SETTING) String clientId,
       @Nullable @Named(OIDC_PROVIDER_SETTING) String oidcProvider,
       @Named(USE_NONCE_SETTING) boolean useNonce,
       @Nullable @Named(OSO_ENDPOINT_SETTING) String osoEndpoint,
       @Nullable @Named(GITHUB_ENDPOINT_SETTING) String gitHubEndpoint) {
 
+    if (serverURL == null && oidcProvider == null) {
+        LOG.error("Either the '" + AUTH_SERVER_URL_SETTING + "' or '" + OIDC_PROVIDER_SETTING + "' property should be set");
+        this.settings = Collections.unmodifiableMap(new HashMap<String, String>());
```

Comment:
just throw ```throw new RuntimeExeception("Either the '" + AUTH_SERVER_URL_SETTING + "' or '" + OIDC_PROVIDER_SETTING + "' property should be set")```

Reply:
or this an acceptable mode or not? maybe ```Collections.emptyMap()```?

Reply:
if it's acceptable why log.error?

Reply:
Fixed in commit https://github.com/eclipse/che/pull/8650/commits/e1e40012b332ef88847ba20c15d4145fe85a353c